meetings/2023-04-19.md

```diff
@@ -0,0 +1,131 @@
+* Join queue to talk.
+* Topics can be proposed at the bottom of the agenda to be discussed as time allows. Make it known if a topic is urgent or cannot be postponed.
+
+### Participation and Code of Conduct
+* [Join the W3C Solid Community Group](https://www.w3.org/community/solid/join), [W3C Account Request](http://www.w3.org/accounts/request), [W3C Community Contributor License Agreement](https://www.w3.org/community/about/agreements/cla/).
+* [Solid Code of Conduct](https://github.com/solid/process/blob/main/code-of-conduct.md), [Positive Work Environment at W3C: Code of Ethics and Professional Conduct](https://www.w3.org/Consortium/cepc/)
+* Operating principle for effective participation is to allow access across disabilities, across country borders, and across time. Feedback on tooling and meeting timing is welcome.
+* If this is your first time, welcome! please introduce yourself.
+
+
+### Scribes
+* Laurens Debackere
+
+### Introductions
+* name: text
+
+---
+
+
+## Topics
+
+### Overview of today's agenda
+
+* SC: Today's meeting will focus on the Solid WG Charter that follows 2023-03-22 meeting topic for reviews: <https://github.com/solid/specification/blob/main/meetings/2023-03-22.md#solid-wg>. See meeting minutes since 2023-03-29 correcting and improving the charter.
+* SC: In previous meetings, we've agreed that we will prepare issues/PRs until we have agreement within the CG; upon agreement, PAC will be assigned to merge them.
+* SC: Questions or suggestions for today's agenda?
+* SC: First open PRs then issues:
+
+
+### fixed spelling and such
+URL: https://github.com/solid/solid-wg-charter/pull/27
+
+* SC: Group agreed that this PR can be merged. Pending PAC's merge.
+* SC: PR's will be brought to a point where there is consensus in the community. Pierre-Antoine will then bring this forward.
+* SC: Some minor remarks on the date format (not necessarily important).
+* PA: In favor of ISO format for dates.
+* SC: Are there any objects to change the date format? If not, SC asks Ted to update the PR to update the document to the ISO format.
+
+
+### Clarify normative, tentative, new Deliverables
+URL: https://github.com/solid/solid-wg-charter/pull/28
+
+* SC: The PR was updated to mention the role or involvement of Solid CG with respect to getting the deliverables into the WG. Still an open question whether, for example, all the normative references of the Solid Protocol should be Working Draft for the Solid Protocol to transition to Candidate Recommendation. We don't have sufficient guidelines, for example, in the context of non-W3C specs such as IETF specifications (RFCs), on what status the normative references should be, perhaps Pierre-Antoine can clarify on this.
+* PA: Is the problem that existing normative refs in current specification? This would be an item for the WG to address.
```

Comment:
```suggestion
* SC: The PR was updated to mention the role or involvement of Solid CG with respect to getting the deliverables into the WG. Still an open question whether, for example, all the normative references of the Solid Protocol should be Working Draft for the Solid Protocol to transition to Candidate Recommendation. We don't have sufficient guidelines, for example, in the context of non-W3C specs such as IETF specifications (RFCs), on what status the normative references should be. Perhaps Pierre-Antoine can clarify on this.
* PA: Is the problem with existing normative refs in current specification? This would be an item for the WG to address.
```